.github/workflows/PythonCI.yml

```diff
@@ -0,0 +1,54 @@
+name: Python CI with pytest and pylint
+
+on: ["push", "pull_request"]
+
+jobs:
```

Comment:
Looks pretty confusing but if it does the job then that's good, they did seem to to take a while to run though. not sure what is a good normal time for it though

Reply:
Yeah that's just the downside of how many packages there are to install, around 4-5 minutes is standard for when I've been running it. There apparently is a way to cache the venv with the packages installed, but it looks pretty complicated to setup and probably isn't worth the headaches it will cause

Reply:
Yep, fair enough. If we don't have a limit on github action use time. Then no need to worry but if we do it might be worth looking at for main Dicom